src/templates/css/styles.css

```diff
@@ -17,6 +18,10 @@ figure {
   width: 0;
 }
 
+ul {
+  list-style-type: none;
```

Comment:
This sets the default for all lists, which is something I would like to avoid. If we remove the bullet, we should align the list items with the surrounding text.